src/index.html

```diff
@@ -1,17 +1,142 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div>
+          <label>Surname:
+            <input
+              type="text"
+              autocomplete="off"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>Name:
+            <input
+              type="text"
+              autocomplete="off"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>How old are You?
+            <input
+              type="number"
+              min="1"
+              max="100"
+              value="12"
+              class="inputBlock"
+            >
+          </label>
+        </div>
+        <div>
+          <label>Full date of birth:
+            <input type="date" class="inputBlock">
```

Comment:
```suggestion
        <div>
          <label>
            Full date of birth:
            <input type="date" class="inputBlock">
```
and better to set text on new line - fix everywhere